lib/store/accounts/user.ex

```diff
@@ -10,7 +10,7 @@ defmodule Store.Accounts.User do
     field :role, Ecto.Enum, values: @roles
     field :hashed_password, :string, redact: true
     field :confirmed_at, :naive_datetime
-
+    has_one :orders, Store.Inventory.Order
```

Comment:
```suggestion
    has_many :orders, Store.Inventory.Order, join_through: Store.Inventory.OrderProducts
```

Maybe add an alias to ```Store.Inventory``` .

A user can have multiple orders. There will only be one draft order, but there can be many that are already delivered (you will not be deleting those from the database), and some waiting to be collected, for example